src/Serilog.Sinks.MSSqlServer/Extensions/ExceptionExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Text;
+
+namespace Serilog.Sinks.MSSqlServer.Extensions
+{
+    internal static class ExceptionExtensions
+    {
+        public static string ToMessageAndCompleteStackTrace(this Exception exception)
+        {
+            var s = new StringBuilder();
+            while (exception != null)
+            {
+                s.AppendLine("Exception type: " + exception.GetType().FullName);
+                s.AppendLine("Message: " + exception.Message);
+                s.AppendLine();
+                exception = exception.InnerException;
+            }
+            return s.ToString();
+        }
```

Comment:
Complete stack trace? Where's the stack trace? Some of the call sites used to just write `Message` (2) others had `ToString` (2) so included the stack trace (and inner exceptions).

If you don't want the stack trace, then maybe still following `Exception.ToString`'s formatting for exception type, message and indentation might be worthwhile.

The SelfLog is pretty detailed as is, I'd have just made them all have `Exception.ToString`.

```
System.Exception: Third thrown
 ---> System.Exception: Second thrown
 ---> System.Exception: First thrown
   at Program.Main()
   --- End of inner exception stack trace ---
   at Program.Main()
   --- End of inner exception stack trace ---
   at Program.Main()
```

Reply:
Thanks @jonorossi! You're completely right. It is much more useful to just use Exception.ToString() in all places instead of manually iterating the nested exceptions and including them. I will close this PR and create a new one where we use ToString().

P.S.:
The name part "Stack" in the exception function is misleading. The original reporter who thankfully created the linked issue suggested this nomenclature and I guess with stack he did not actually mean the programs call stack when the exception was thrown but the full list of inner exceptions, which are two different things entirely. I kind of noticed this misleading name when creating the PR but did not bother to change it.